site-content/source/modules/ROOT/pages/blog/Apache-Cassandra-Changelog-11-January-2022.adoc

```diff
@@ -0,0 +1,114 @@
+
+To stay up-to-date, we recommend joining the Cassandra  https://cassandra.apache.org/community/[mailing list^].
+
+== Community Notes
+
+_Updates on Cassandra Enhancement Proposals (CEPs), how to contribute, and other community activities._ 
+
+_Are you new to the project? We have established a https://issues.apache.org/jira/secure/RapidBoard.jspa?rapidView=484[New Release tracking Kanboard^] and a https://issues.apache.org/jira/secure/RapidBoard.jspa?rapidView=484&quickFilter=2162&quickFilter=2160[“Starter Tickets”^] quick label that corresponds to our Low Hanging Fruit status. Any of these tickets should be of appropriate complexity for someone new to the project to tackle. You can also reach out in the #cassandra-dev Slack channel on the-asf.slack.com and ask for assistance from any of the 13 designated Cassandra mentors!
+
+
+Read PMC member Josh McKenzie’s https://lists.apache.org/thread/9rzwl3s2fr734sdc5jynbxz3z4vht3df[latest bi-weekly update^] for ongoing discussions and the latest on ticket progress.
+
+===  Advisory
+
+Apache Cassandra is _not_ vulnerable to the log4shell vulnerability, as you can see in Jira ticket, https://issues.apache.org/jira/browse/CASSANDRA-5883[CASSANDRA-5883^], Apache Cassandra never chose to use log4j2 preferring logback instead.
+
+=== Announced
+
+During December we ran an https://issues.apache.org/jira/issues/?jql=project%20%3D%20CASSANDRA%20AND%20labels%20%3D%20AdventCalendar2021%20[Advent Calendar (Twitter initiative^] of new tickets to get started on. This was a great way for new project contributors to pick up hand-curated tickets to dive into.
+
+image::blog/advent-calendar.png[Apache Cassandra’s advent calendar campaign on Twitter]
+
+=== Announced
+
+It may be cold outside, but we’re warming ourselves up with thoughts of a great Google Summer of Code, and there are some exciting changes this year! Read https://lists.apache.org/thread/58v2bvfzwtfgqdx90qmm4tmyoqzsgtn4[Paulo Motta’s post^] on the mailing list. 
+
+=== Announced
+
+Ubuntu 21.10 includes Apache Cassandra https://www.devopsdigest.com/canonical-releases-ubuntu-2110[“packaged as a snap, enabling DataOps teams to set up a Cassandra cluster”^], and the latest LTS Docker Images now include Apache Cassandra v4.
+
+=== Announced
+
+Apple indicates support and encourages further contribution to the Apache Cassandra community with a https://opensource.apple.com/projects/[new online portal^] covering all its open source projects and contributions.
+
+=== Added
+
+The Project Management Committee (PMC) is pleased to announce that *Sumanth Pasupuleti* has been invited to https://lists.apache.org/thread/kvsvltom5p79bk2r1j52sxl3y3qjl71w[become a committer^], and he has accepted! Thank you for all your contributions Sumanth, and congratulations! 👏
+
+=== Added
+
+An https://cwiki.apache.org/confluence/display/CASSANDRA/Roadmap[up-to-date roadmap^] for the project has been added to the wiki.
```

Comment:
I'd like to suggest using the permanent URLs (permalink) for cwiki pages so that when the wiki titles get changed, the permalink will continue to work.

```suggestion
An https://cwiki.apache.org/confluence/x/gYvkCw[up-to-date roadmap^] for the project has been added to the wiki.
```